content/controllers.md

```diff
@@ -256,10 +256,10 @@ Returned values will override any arguments passed to the `@Redirect()` decorato
 
 ```typescript
 @Get('docs')
-@Redirect('https://docs.nestjs.com', 302)
+@Redirect('', 302)
 getDocs(@Query('version') version) {
   if (version && version === '5') {
-    return { url: 'https://docs.nestjs.com/v5/' };
+    return { url: '/v5/' };
   }
 }
 ```
```

Comment:
Does this code snippet also need to be changed?

Reply:
nop, my bad. Thanks

Reply:
you're welcome, it's nothing 👍